python_transport/wirepas_gateway/transport_service.py

```diff
@@ -609,6 +627,12 @@ def main():
 
     try:
         debug_level = os.environ["DEBUG_LEVEL"]
+        print("Deprecated environment variable it will be removed with >2.x")
```

Comment:
Maybe you could name the environment variable in the message and also give the new variable name

Reply:
Good point

Reply:
done